src/Plugin_Command.php

```
@@ -1128,6 +1128,13 @@ protected function get_wporg_data( $plugin_name ) {
 				if ( ! $this->check_wporg['last_updated'] ) {
 					return $data; // The plugin is active on .org, but we don't need the date.
 				}
+				// The plugins API already reports when the plugin was last updated, so use
+				// that instead of also scraping the trac log, which is rate-limited and
+				// otherwise unnecessary here.
+				if ( ! empty( $plugin_data['last_updated'] ) ) {
+					$data['last_updated'] = $this->format_wporg_last_updated( strtotime( $plugin_data['last_updated'] ) ?: null );
```

PHPStan (2.2.8)

[error] Parameter `#1` $datetime of function strtotime expects string, mixed given.


(argument.type)